src/client/src/containers/DependencyInfo/styles.module.css

```diff
@@ -8,6 +8,10 @@
   text-decoration: none;
 }
 
+.dependencyContainer:focus {
+  outline: 1px solid var(--vscode-contrastActiveBorder);
+}
+
```

Comment:
What does this style do?

Reply:
It adds the orange border when u navigate to it using tab (we use this css in all the navigable components).